test/acceptance/framework/config/config.go

```diff
@@ -87,7 +88,16 @@ func (t *TestConfig) entImage() (string, error) {
 		return "", err
 	}
 
-	return fmt.Sprintf("hashicorp/consul-enterprise:%s-ent", chartMap["appVersion"]), nil
+	appVersion := chartMap["appVersion"].(string)
+	var preRelease string
+	// Handle versions like 1.9.0-rc1.
```

Comment:
Do all our pre-releases share the semver `x.y.z-rc*` ?

Reply:
Yeah afaik. We have `-rc<n>` and `-beta<n>`